source/initial-dns-seedlist-discovery.rst

```diff
@@ -0,0 +1,247 @@
+﻿.. role:: javascript(code)
+  :language: javascript
+
+==============================
+Initial DNS Seedlist discovery
+==============================
+
+:Spec-ticket: SPEC-878
+:Title: Initial DNS Seedlist discovery
+:Authors: Derick Rethans
+:Status: Draft
+:Type: Standards
+:Last Modified: 2017-08-29
+:Version: 0.3
+:Spec Lead: Matt Broadstone
+:Advisory Group: A. Jesse Jiryu Davis
+:Approver(s): Bernie Hackett (2017-08-17), David Golden (2017-08-29), Jeff Yemin (2017-08-22), Matt Broadstone (2017-08-24), A. Jesse Jiryu Davis (2017-08-14)
```

Comment:
Escape `\A.` here as well.

Reply:
Interestingly enough, this case will not cause the list to happen. I originally solved the issue with Jesse's `A.` by simply moving him down the list of approvers in change streams

Reply:
Ah, it's likely because his name is no longer the first term in the table cell.